### ISSUE TITLE
[Discussion] Datasets with Identified Errors or Issues

### ISSUE THREAD

**Issue by YanchunHe** [NorESMhub/noresm2cmor#320]
(reserved space for future updates).

**Comment by YanchunHe** [comment]
Hi,

I get to know some would create a page/documentation to list the identified errors in the published datasets in addition to ESDoc Errta, to make for accessible to domestic users for NorESM output for CMIP5/6.

Here I initiate this discussion, and any ideas and comments are appreciated!

### Option 1, use markdown in-line table in Github Issue or Wiki page

(I wold prefer Pinned Github Issue page).

|Variable | Table ID | Issue | Experiment | Model | null |
|--           |--           |--           |--           |--              | ---  |
|   fLuc             |   Lmon        |   https://github.com/NorESMhub/noresm2cmor/issues/319 |   historical |   NorESM2-LM |
|   fHarvest      |   Lmon        |   https://github.com/NorESMhub/noresm2cmor/issues/319 |   historical |   NorESM2-LM |
|   wap             |   6hrPlev     |   https://github.com/NorESMhub/noresm2cmor/issues/316 |   ssp585, * |   NorESM2-LM |
|   co2              |   AERmon   |   https://github.com/NorESMhub/noresm2cmor/issues/315 |   piControl |   NorESM2-LM |

### Option 2, html table with search/filter for variable

I can create a html page with search/filter, an example:
http://ns2345k.web.sigma2.no/people/yanchun/tmp/issues.html

However, such html code can not be embed in Github Issue or Wiki pages. I can't find proper place to hold the html file or maybe create with an online spread sheet and provide a link in this repository (e.g., in README at the front page)?

**Comment by TomasTorsvik** [comment]
Hi @YanchunHe ,

if it is enough with a simple table, I think it would be best to have it integrated on the github platform and make use of the existing Wiki page. I would prefer this solution over a pinned github issue page, since the wiki can be edited by everyone, whereas in an issue we can only edit our own comments. Maybe you can elaborate on why you prefer to have it as an issue?

At the moment I don't think we need a search/filter option ('ctrl + f' in a web page works fine for short lists), but I see the point of having this option if the list grows into the hundreds. At the moment I see 31 NCC-related issued on the ES-Doc Errata page. Do we anticipate to have a significantly higher number for the list we are discussing now?

**Comment by YanchunHe** [comment]
Thanks, @TomasTorsvik for your good suggestions!

>  I would prefer this solution over a pinned github issue page, since the wiki can be edited by everyone, whereas in an issue we can only edit our own comments. 

This is a good point if all the users would like to add an entry for identified error. My thinking is that the GitHub Issues is always more 'visible' to people, so that they can easily spot there is a compiled list of datasets with errors.

> At the moment I don't think we need a search/filter option ('ctrl + f' in a web page works fine for short lists), but I see the point of having this option if the list grows into the hundreds. At the moment I see 31 NCC-related issued on the ES-Doc Errata page. Do we anticipate to have a significantly higher number for the list we are discussing now?

We can use a static table in wiki. As it would stay clean in the wiki format, so that it should be fine with limited amount of entries.

**Comment by adagj** [comment]
Hi @YanchunHe !
Great that you are looking into this, and a table sounds like a good idea. I think one challenge is that issues concerning data errors have been closed without actually fixing the error or providing the information elsewhere; e.g the sea ice lat/lon boundaries. Can we also add closed issues to the table? Then it may be a quite extensive table after a while.

**Comment by YanchunHe** [comment]
Yes, @adagj we can include the previously closed issues. I was thinking to add a 'status' column to this table, something as 'fixed', 'closed', "will not fix", "errata reported", or something similar.

If considering its expansion and to make it searchable, I would then recommend a github-pages, or even simply just use an external spreadsheet, such as google sheet (everyone can edit or only those with granted permission?)
